code/src/java/pcgen/output/model/CDOMObjectModel.java

```diff
@@ -75,14 +74,29 @@ public CDOMObjectModel(CDOMObjectWrapperInfo info, CDOMObject cdo)
 	@Override
 	public TemplateModel get(String key) throws TemplateModelException
 	{
-		OutputActor<CDOMObject> actor = info.get(key);
+		//TODO may not be entirely correct - really about ... what? formula scope??
```

Comment:
oops.  I will fix this.  It was addressed.  The use case that was a problem was something like SubClass (hence my comment about formula scope - which is actually not right either), but the solution was CDOMWrapperInfoFacet.getActor() will "fall up" - so SubClass can use PCClass and both can use CDOMObject.class (just like the tokens "fall up" to allow global tokens.